config.toml

```diff
@@ -0,0 +1,100 @@
+# Site settings
+baseurl = "/"
+languageCode = "en-us"
+title = "Okkur - Syna"
+theme = "syna"
+themesDir = "themes"
+enableGitInfo = true
+version = "0.11.0"
+
+[frontmatter]
+date = ["date", "lastmod"]
+lastmod = ["lastmod", ":git", "date"]
+
+# Google Analytics tracking
+googleAnalytics = ""
+
+[params]
+  name = "Okkur Labs"
+  description = "Open Source Theme for your next project"
+
+  [params.style]
+    #background = "secondary"
+
+  # Theme customizations
+  # Assets relative to 'static/'
+  [params.custom]
+    favicon = "favicon.ico"
+    favicon_ico_sizes = "16x16 32x32"
+    favicon_png = "favicon.png"
+    favicon_svg = "favicon.svg"
+    css = []
+    js  = []
+
+  # Piwik tracking
+  [params.analytics.piwik]
+    url = "https://use.okkur.net"
+    id = "13"
+    domain = "syna-demo.okkur.io" # Track subdomains via "*.example.com"
```

Comment:
Let's remove our specific piwik config aka make the values empty and comment them out additionally.